client/verta/verta/runtime.py

```diff
@@ -118,8 +120,9 @@ def log(key: str, value: Any) -> None:
         If `validate` was set to ``True`` on the active :class:`context`, and
         `value` is not a JSON-serializable type.
     ValueError
-        If `key` provided contains non-alphanumeric characters.  Dashes `-`
-        and underscores `_` are permitted.
+        If `key` provided contains non-alphanumeric characters (Dashes `-`
+        and underscores `_` are permitted) or the key already exists and the
```

Comment:
A comma here will help a bit with clarity.

```suggestion
        and underscores `_` are permitted), or the key already exists and the
```

Reply:
Done